js/chatbot/shared/ChatBot.svelte

```diff
@@ -562,51 +534,63 @@
 		display: none;
 	}
 
-	.suggestions {
-		padding-top: calc(var(--spacing-xxl) * 10);
-		margin-bottom: calc(var(--spacing-xxl) * -10);
+	.placeholder-content {
 		display: flex;
-		flex-direction: row;
+		flex-direction: column;
+		height: 100%;
+	}
+
+	.placeholder {
+		align-items: center;
+		display: flex;
+		justify-content: center;
+		height: 100%;
+		flex-grow: 1;
 	}
 
-	.suggestions :global(img) {
```

Comment:
Adding this back, otherwise clicking on an image in examples makes it fullscreen.